scripts/uploadToS3.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+
+const chalk = require("chalk")
+const glob = require("glob")
+const mime = require("mime")
+const path = require("path")
+const s3 = require("s3")
+const { last } = require("lodash")
+
+const options = {
+  files: path.join(__dirname, "../**/public/**"),
+  root: "public",
+  key: process.env.S3_KEY,
+  secret: process.env.S3_SECRET,
+  bucket: "artsy-chris-test-bucket", // process.env.S3_BUCKET,  // artsy-force-$DEPLOY_ENV
+  cdnUrl: process.env.CDN_URL,
+}
+
+// See: https://www.npmjs.com/package/s3 for more options
+const client = s3.createClient({
+  s3Options: {
+    accessKeyId: options.key,
+    secretAccessKey: options.secret,
+  },
+})
+
+const files = glob.sync(options.files, { nodir: true }).filter(file => {
+  return !file.match("node_modules")
```

Comment:
Not important, but can't this be done by adding an exclusion clause to the glob pattern?